src/hitchhiker/tree/messaging.cljc

```diff
@@ -0,0 +1,252 @@
+(ns hitchhiker.tree.messaging
```

Comment:
Unfortunately that breaks the tests... Anyhow once the changes to `core.cljc` and `messaging.cljc` are ok from your side I can remove them again. They pass atm. for me.